pyccl/tests/test_tkkssc.py

```diff
@@ -249,3 +250,146 @@ def test_tkkssc_counterterms_gc(kwargs):
 
     assert np.abs((tk_nogc_12 - tkc12) / tk_gc_12 - 1).max() < 1e-5
     assert np.abs((tk_nogc_34 - tkc34) / tk_gc_34 - 1).max() < 1e-5
+
+
+@pytest.mark.parametrize('kwargs', [{f'is_number_counts{i+1}': nc[i] for i in
+                                     range(4)} for nc in
+                                    itertools.product([True, False],
+                                                      repeat=4)])
+def test_tkkssc_linear_bias(kwargs):
+    hmc = ccl.halos.HMCalculator(COSMO, HMF, HBF, mass_def=M200,
+                                 nlog10M=2)
+    k_arr = KK
+    a_arr = np.array([0.3, 0.5, 0.7, 1.0])
+
+    # Tk's exact version
+    prof = ccl.halos.HaloProfileNFW(ccl.halos.ConcentrationDuffy08(M200),
+                                    fourier_analytic=True)
+    bias1 = 1
+    bias2 = 2
+    bias3 = 3
+    bias4 = 4
```

Comment:
can you start from 2 instead of 1? That way all biases are !=1 (in case bias1=1 is hiding a bug somewhere)